testing/src/main/java/org/conscrypt/TestUtils.java

```diff
@@ -33,16 +36,19 @@
 import javax.net.ssl.SSLException;
 import javax.net.ssl.SSLServerSocketFactory;
 import javax.net.ssl.SSLSocketFactory;
+import libcore.io.Streams;
 import libcore.java.security.TestKeyStore;
 
 /**
  * Utility methods to support testing.
  */
-public final class TestUtil {
+public final class TestUtils {
+    public static final Charset UTF_8 = Charset.forName("UTF-8");
```

Comment:
This one I actually think we should keep since it's used by other tests which are Java >= 1.6.